packages/testcontainers/src/containers/RegTestContainer/index.ts

```diff
@@ -42,6 +42,7 @@ export class RegTestContainer extends DeFiDContainer {
       '-fortcanningepilogueheight=15',
       '-grandcentralheight=16',
       '-grandcentralepilogueheight=17',
+      '-nextnetworkupgradeheight=18',
```

Comment:
yea can help need update here as well? (in case :pray: )
[apps/whale-api/docker-compose.yml](https://github.com/BirthdayResearch/jellyfishsdk/blob/6dbe03fadad166afaf5c3d4853675c1682c5d33e/apps/whale-api/docker-compose.yml#L42)

Reply:
@canonbrother @fullstackninja864 should we change this to changi?